### PR TITLE
Final Prize Rules

### PR DESCRIPTION
**Pull request by MarkBennett**
Also general clean up of rules.

I've split the FAQ and Submission sections into their own pages as it was getting to long and we might want to be able to link to them directly.

TODO:
* [x] Fix typos
* [x] Clean up grammar and flow
* [x] Answer Aaron's question about attribution
* [x] Link rules, FAQ, and submission up to menu in `default` layout
* [x] Add the rules abstract to the home page with link to full rules
* [x] Clean up submission page

### CODE REVIEW

File: rules.md [hunk 1]
```diff
@@ -3,104 +3,131 @@ layout: default
...
-    *   Instructions for interacting with your app (if not included in the app itself).
-    *   An email address that judges can use to follow up with the team should they have questions or concerns. This will not be shared with the public, or with voters.
-    *   For open-source projects, we encourage a link to the source code repository (e.g. GitHub, GitLab, Bitbucket, etc.).
-*   Your submission does not need to be complete, but you must be able to demonstrate it.
-*   Teams may submit their projects for multiple weekly considerations; however, new progress must be demonstrated leading up to each submission.
-*   Your solution does not need to be open-source, but it must be available (upon request) for scrutiny by organizers and judges.
-*   In the spirit of hackathons, only apps that were started **on or after March 16, 00:00 UTC** will be considered for prizing.
-*   It bears repeating: organizers and judges reserve the right to disqualify entries and winners who violate the [Code of Conduct](code-of-conduct.html) and [Submission](#submission) rules as outlined on this website.
+1. Supporting Crisis Response
+2. (Mis)Understanding The Pandemic
```
Comment (ajyong) [inline rules.md]: My knee-jerk reaction to the "(Mis)" part was confusion.  I think grammatically, "Misunderstanding of the pandemic" is different from "Misunderstanding the pandemic"; the latter is how it's currently read in my mind.

It might be too wordy, but something along the lines of "(Mis)information about the Pandemic" reads less ambiguously.

File: rules.md [hunk 2]
```diff
@@ -3,104 +3,131 @@ layout: default
...
 
-*   A "week" for prizing purposes **starts Monday, 00:00 UTC** and ends **Sunday, 23:59 UTC**.
-*   Submissions **closes** every Sunday, 23:59 UTC of that "week".
-*   Teams can submit their project for consideration through this form _(this is coming soon!)_
-*   Once organizers have compiled submissions, voting is by **popular vote** and is open to the **general public**.
-*   Following submission closure, voting starts **Monday, 03:00 UTC** and lasts 24 hours (ends Tuesday, 03:00 UTC). This is a great opportunity to promote your project and CODEVID-19 to friends, family, and your network!
-    *   **Addendum:** for the First Responder's prize, we've moved end of voting to **Tuesday, 14:00 UTC**.
-*   The winner will be announced by **Tuesday, 20:00 UTC**.
-*   Prizes of monetary value will be distributed to winning teams electronically.
+#### (Mis)Understanding The Pandemic
```
Comment (ajyong) [inline rules.md]: Just in case, see previous comment.

File: rules.md [hunk 3]
```diff
@@ -3,104 +3,131 @@ layout: default
 title: CODEVID-19 Rules
```
Comment (ajyong) [inline rules.md]: I was able to nest the [privacy policy](https://github.com/devedmonton/codevid19.com/blob/master/policies/privacy-policy.html) under a "Policies" folder.  I think it keeps our repo looking tidier if we put these files in there as well.

File: rules.md [hunk 4]
```diff
@@ -3,104 +3,131 @@ layout: default
...
+### Your Team
 
-#### Okay, so what's the catch?
+Teams of up to 5 people are welcome to participate. For legal reasons we require that all team members be 13 years or older as of May 16th at 00:00 UTC if they wish to participate.
 
-Participants and volunteers must abide by and uphold the [Code of Conduct](code-of-conduct.html), [Rules](#rules-submission-faq) and generally be an upstanding human being.
+### Prizes
 
-#### If I see organizers give out a time/deadline, what time zone is it in?
+Prizes of monetary value will be distributed to winning teams electronically. All other prizes will be disttributed in the manner deamed most appropriate by organizers and sponsors.
```
Comment (ajyong) [inline rules.md]: `s/disttributed/distributed/g` :P

Should we also be mentioning _when_ teams should expect an organizer to begin correspondence for prize awarding purposes?  It could be different for the weekly versus final prizing.

File: rules.md [hunk 5]
```diff
@@ -3,104 +3,131 @@ layout: default
...
 
-Double-check with them, but we are running in UTC. Check [here](https://www.timeanddate.com/worldclock/) to see what that is relative to your local time.
+Some sponsors may be given the opportunity award prizes during the course of the competition. These prizes will be communicated and awarded by sponsors in the manner they sit fit, and subject to the rules and veto of the event organizers.
```
Comment (ajyong) [inline rules.md]: "they sit fit" -> "they see fit", I make these mistakes all the time and it sucks because spellcheckers can't catch 'em :(

File: rules.md (hunk 3 above)
Comment (MarkBennett) [reply rules.md]: Good call. I'll move these in there.

File: rules.md (hunk 4 above)
Comment (MarkBennett) [reply rules.md]: Nah. I don't think there's a value in specifying this other than to get us in trouble.

**Comment by kdehaan** [comment]
(wrong button, sorry!)

File: rules.md [hunk 6]
```diff
@@ -3,104 +3,131 @@ layout: default
...
 
-Volunteers are busy working on several items right now, and we're sorry we don't have a pre-compiled list for you. That said, there's been several posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join!
+We believe faster, more effective and more available solutions are preferred to slower, less useful and limited ones.
 
-#### Will all projects be judged? Can I opt not to have my project submitted?
+Therefore we will be judging projects on three main criteria:
 
-Absolutely! We ask that teams who want to compete for prizing to submit their projects on their own.
+1. **Effectiveness** - How well does this project address the problem it's attempting to solve?
+2. **Acessability** - How easily are people are able to access this project?
```
Comment (kdehaan) [inline rules.md]: Acessability -> Accessibility

File: rules.md (hunk 4 above)
Comment (kdehaan) [inline rules.md]: disttributed -> distributed

File: rules.md [hunk 7]
```diff
@@ -3,104 +3,131 @@ layout: default
...
 
-Rules, Submissions & FAQ
-------------------------
+## Abstract
 
-### Rules
+The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when judging starts. Following judging final prizes are awarded.
 
-#### How big can teams be?
+Projects compete in four problem areas, and are evaluated on effectiveness, accessability, and impact.
```
Comment (kdehaan) [inline rules.md]: accessability -> accessibility

File: rules.md (hunk 4 above)
Comment (kdehaan) [reply rules.md]: deamed -> deemed

File: rules.md [hunk 8]
```diff
@@ -3,104 +3,131 @@ layout: default
...
+
+Teams may also be asked to confidentially share source code and project details with judges so that they might better evaluate a projects merits.
+
+Failure to communicate or supply judges with access to a project could impact a projects success in the competition.
+
+### Intellectual Property
+
+All teams will continue to own the intellectual rights to their work and projects during and after this competition.
+
+Projects are encouraged to open source their code where applicable, but are not required to do so. During judging teams may be asked to provide judges access to source code to verify technical and other claims. Failure to do so could disadvantage or disqualify teams during judging, at the discretion of the judging commitee and organizers.
```
Comment (kdehaan) [inline rules.md]: commitee -> committee

File: rules.md [hunk 9]
```diff
@@ -3,104 +3,131 @@ layout: default
...
+
+Teams are welcome to use someone else's assets (e.g. libraries, frameworks, templates, source code, images, etc.) but are required to complie with the licensese for these assets. Teams may be asked to supply proof they have done so. Teams violating license and intellectual property rights of others my be disqualified and forfeit prizes awarded them.
```
Comment (kdehaan) [inline rules.md]: complie -> comply 
licensese -> licenses

File: rules.md [hunk 10]
```diff
@@ -3,104 +3,131 @@ layout: default
...
+
+### Media Consent
+
+From time-to-time projects and other information about the Hackathon will be shared publicly via social and other media. We respect participants right to privacy and will not share personal information without prior consent.
+
+### Communication Consent and Responsibilities
+
+By participating participants agree to give organizers permission to contact them with information and communication relevant to CODEVID-19 via Slack, email, or other methods deemed appropriate by event organizers. This consent may be withdrawn at anytime be contacting contact@codevid19.com, or using the unsubscribe feature in the mailing list.
+
+Participants understand that by withdrawing their consent to communicate with organizers they will be putting themselves, their team, and their project at a competative disadvantage. Organizers cannot be held responsonsible for participants receiving information supplied via email or Slack. It's participants resonsponsibiltiy to regularly check the website or Slack for updates should they choose to withdraw consent.
```
Comment (kdehaan) [inline rules.md]: competative -> competitive 
responsonsible -> responsible 
resonsponsibiltiy -> responsibility

File: rules.md [hunk 11]
```diff
@@ -3,104 +3,131 @@ layout: default
...
+
+All participants, organizers, sponsors and contributors agree to follow [the CODEVID-19 Code of Conduct](code-of-conduct.html) and understand that failure to do so could result in consequences up to and including elimination and exclusion from competition, Slack, and other resources provided along with forfiture of any prize winnings or consideration provided by CODEVID-19, the Dev Edmonton Society, and associated parties. Participants understand that enforcing and interpretting the Code of Conduct is done at the sole discretion of the CODEVID-19 organizing volunteers. Questions regarding the enforcement or interpretaion of the Code of Conduct should be directed to conduct@codevid19.com.
```
Comment (kdehaan) [inline rules.md]: forfiture -> forfeiture 
interpretting -> interpreting 
interpretaion -> interpretation

File: rules.md [hunk 12]
```diff
@@ -3,104 +3,131 @@ layout: default
...
+
+Participants are welcome to use any language they deem most appropriate for their application, however CODEVID-19 reserves the right to only accept submssions in English, or where an English translation is provided, should a project wish to be considered for prizes.
+
+To participate, team members must be at least 13 years of ago at the time the competition started on March 16th at 00:00 UTC. To be considered for prizes, teams must include no more than 5 people.
+
+Prizes amounts and types will be determined at the discretion of organizers and event sponsors, and distriputed in a manner they see fit.
```
Comment (kdehaan) [inline rules.md]: distriputed -> distributed

File: rules.md [hunk 13]
```diff
@@ -3,104 +3,131 @@ layout: default
...
+
+All participants, organizers, sponsors and contributors agree to follow [the CODEVID-19 Code of Conduct](code-of-conduct.html) and understand that failure to do so could result in consequences up to and including elimination and exclusion from competition, Slack, and other resources provided along with forfiture of any prize winnings or consideration provided by CODEVID-19, the Dev Edmonton Society, and associated parties. Participants understand that enforcing and interpretting the Code of Conduct is done at the sole discretion of the CODEVID-19 organizing volunteers. Questions regarding the enforcement or interpretaion of the Code of Conduct should be directed to conduct@codevid19.com.
+
+Participants are welcome to use any language they deem most appropriate for their application, however CODEVID-19 reserves the right to only accept submssions in English, or where an English translation is provided, should a project wish to be considered for prizes.
```
Comment (kdehaan) [inline rules.md]: submssions -> submissions

File: faq.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+Volunteers are busy working on several items right now, and we're sorry we don't have a pre-compiled list for you. That said, there's been several posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join!
+
+#### Will all projects be judged? Can I opt not to have my project submitted?
+
+Absolutely! We ask that teams who want to compete for prizing to submit their projects on their own.
+
+#### What's the cost of entry?
+
+This hackathon is free to enter and particpate in! We thank our sponsors for supporting our endeavours.
```
Comment (kdehaan) [inline faq.md]: particpate -> participate

**Review by kdehaan — CHANGES_REQUESTED** [review]

**Comment by MarkBennett** [comment]
Alright, I've addressed everyone's feedback. Please take one final look then I'll ship these.

Note it came up that we should add a calendar of significant events somewhere. Will make a separate PR for that.

/cc @ajyong @kdehaan

File: policies/faq.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+
+That's fine, however only projects that were started **on or after March 16, 00:00 UTC** can be submitted for prizing considerations. See [Submissions](#submission) for more details.
+
+#### When do sign-ups start? When do they end?
+
+Sign-ups have started! Register on [FindCollabs](https://findcollabs.com/hackathon/codevid-19-isp21fkqtjupchx7kjed) and [join](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) our [Slack](https://codevid-19.slack.com). We will close sign-ups one week prior to the [final submission date](#submission).
+
+#### Do you have datasets and other resources we could start from?
+
+Volunteers are busy working on several items right now, and we're sorry we don't have a pre-compiled list for you. That said, there's been several posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join!
```
Comment (kdehaan) [inline policies/faq.md]: "olunteers are busy working on several items right now, and we're sorry we don't have a pre-compiled list for you. That said, there's been several posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join!" -> "We're working on a pre-compiled list, but until it's available you can find many tools posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join" (Personal preference, it's still good as it is)

File: policies/rules.md [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+1. Supporting Crisis Response
+2. Understanding The Pandemic
+3. Social Distancing & Isolation 
+4. Scarcity & The Economy
+
+#### Supporting Crisis Response
+
+First responders, healthcare workers and emergency response managers are on the front lines of the pandemic. How can we help to work more effectively, safely, and quickly?
+
+Examples of apps in this catergory might include services to crowd source reporting of crisis information, using machine learning and data science to quickly identify problems to be addressed, or allow more efficient distribution of health care and emergency resources.
```
Comment (kdehaan) [inline policies/rules.md]: catergory -> category

File: policies/rules.md [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+
+#### Understanding The Pandemic
+
+How can we better understand the current state of the pandemic and make informed decisions about the future? How do we evaluate and trust the information around us?
+
+An app in this category might help provide verified answers to questions about COVID-19 and the pandemic, predict if the curve is being flattened, or help people report and filter out mis-information.
```
Comment (kdehaan) [inline policies/rules.md]: mis-information -> misinformation

File: policies/rules.md [hunk 18]
```diff
@@ -0,0 +1,145 @@
...
+
+#### Social Distancing & Isolation
+
+From working at home, assisting the elderly and at need, to entertaining and educating our children few parts of the pandemic more broadly impact society than the Social Isolation and Distancing we're all practicing.
+
+How do we limit the impact and costs associated with Social Isolation and Distancing, and ensure that we're ready to pick up where we left off when we come back together?
+
+Apps in this category might help connect people together, indentify people in distress, facilitate remote working, on assist parents and educators continue classes.
```
Comment (kdehaan) [inline policies/rules.md]: indentify -> identify

File: policies/rules.md [hunk 19]
```diff
@@ -0,0 +1,145 @@
...
+
+### Your Solution
+
+We believe faster, more effective and more available solutions are preferred to slower, less useful and limited ones.
+
+Therefore we will be judging projects on three main criteria:
+
+1. **Effectiveness** - How well does this project address the problem it's attempting to solve?
+2. **Accessibility** - How easily are people are able to access this project?
+3. **Impact** - How many people have benefitted from this project?
```
Comment (kdehaan) [inline policies/rules.md]: benefitted -> benefited

File: policies/rules.md [hunk 20]
```diff
@@ -0,0 +1,145 @@
...
+
+### Media Consent
+
+From time-to-time projects and other information about the Hackathon will be shared publicly via social and other media. We respect participants right to privacy and will not share personal information without prior consent.
+
+### Communication Consent and Responsibilities
+
+By participating participants agree to give organizers permission to contact them with information and communication relevant to CODEVID-19 via Slack, email, or other methods deemed appropriate by event organizers. This consent may be withdrawn at anytime be contacting contact@codevid19.com, or using the unsubscribe feature in the mailing list.
+
+Participants understand that by withdrawing their consent to communicate with organizers they will be putting themselves, their team, and their project at a competitive disadvantage. Organizers cannot be held responsible for participants receiving information supplied via email or Slack. It's participants resonsponsibiltiy to regularly check the website or Slack for updates should they choose to withdraw consent.
```
Comment (kdehaan) [inline policies/rules.md]: resonsponsibiltiy -> responsibility

File: policies/submissions.md [hunk 21]
```diff
@@ -0,0 +1,39 @@
...
+* Teams may submit their projects for multiple weekly considerations; however, new progress must be demonstrated leading up to each submission.
+
+## Final Prize Submission
+
+When submitting for final consideration, teams must also include:
+
+1. A link to a short, 3 minutes or less, pitch and demo that clearly explains:
+    a. the problem being solved
+    b. how to use the app
+    c. any unique features or information pertanent to the judging
```
Comment (kdehaan) [inline policies/submissions.md]: pertanent -> pertinent

**Review by kdehaan — CHANGES_REQUESTED** [review]

File: index.html [hunk 22]
```diff
@@ -16,7 +16,7 @@
         <li><a href="#intro">Welcome</a></li>
         <li><a href="#join-the-fight">Join the Fight</a></li>
         <li>
-          <a href="rules.html">Rules, Submissions &amp; FAQ</a>
+          <a href="#rules-and-faq">Rules & FAQ</a>
```
Comment (ajyong) [inline index.html:19]: Call me old-school but does `&` still need to be replaced with `&amp;` these days to be compatible with all kinds of browsers?  I just got into the habit but that was circa... 2005?

File: index.html [hunk 23]
```diff
@@ -169,6 +169,33 @@ <h3>Sponsors & Partners</h3>
     </div>
   </section>
 
+  <section id="rules-and-faq" class="wrapper style2 fade-up">
+    <div class="inner">
+      <h2>Rules</h2>
+      <p>The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when projects can submit their projects for the final prize consideration. Following judging final prizes are awarded.</p>
```
Comment (ajyong) [inline index.html]: > Following judging final prizes are awarded.

Can be tweaked:

> Final prizes are awarded following a judging period.

Do we have an idea of how long we should be giving judges to do so?  I'm feeling like a week is good.  We should add this in once we know what the time scale will be.

File: index.html [hunk 24]
```diff
@@ -169,6 +169,33 @@ <h3>Sponsors & Partners</h3>
...
+      <strong>
+        <ol class="strong">
+          <li>Supporting Crisis Response</li>
+          <li>Understanding The Pandemic</li>
+          <li>Social Distancing & Isolation</li>
+          <li>Scarcity & The Economy</li>
+        </ol>
+      </strong>
+
+      <p>Each project is evaluated on effectiveness, accessibility, and impact. The judges will award Winner and Runner Up prizes in each problem area. There's also a final Popular Choice prize. <a href="{{ "/policies/rules.html" | relative_url }}">Read the full rules</a> for all the details, and checkout the <a href="{{ "/policies/faq.html" | relative_url }}">Frequently Asked Questions (FAQ)</a>.</p>
```
Comment (ajyong) [inline index.html]: "checkout" should be "check out"

File: policies/rules.md [hunk 25]
```diff
@@ -3,82 +3,88 @@ layout: default
 title: CODEVID-19 Rules
 ---
 
-## Abstract
+## Summary
 
-The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when judging starts. Following judging final prizes are awarded.
+The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when projects can submit their projects for the final prizes. Following judging final prizes are awarded.
```
Comment (ajyong) [inline policies/rules.md]: > when projects can submit their projects for the final prizes

Should that be "when teams can submit their projects"?

Comment (ajyong) [inline policies/rules.md]: The "following judging" grammar thing again.  Gah, GitLab lets me link to other comments, why can't GitHub??

**Comment by MarkBennett** [comment]
Thanks for the spelling catches @kdehaan. I'll make those changes in the AM and post this.

File: policies/rules.md [hunk 26]
```diff
@@ -0,0 +1,145 @@
...
+
+All teams will continue to own the intellectual rights to their work and projects during and after this competition.
+
+Projects are encouraged to open source their code where applicable, but are not required to do so. During judging teams may be asked to provide judges access to source code to verify technical and other claims. Failure to do so could disadvantage or disqualify teams during judging, at the discretion of the judging committee and organizers.
+
+Teams are welcome to use someone else's assets (e.g. libraries, frameworks, templates, source code, images, etc.) but are required to comply with the licenses for these assets. Teams may be asked to supply proof they have done so. Teams violating license and intellectual property rights of others my be disqualified and forfeit prizes awarded them.
+
+### Media Consent
+
+From time-to-time projects and other information about the Hackathon will be shared publicly via social and other media. We respect participants right to privacy and will not share personal information without prior consent.
```
Comment (ajyong) [inline policies/rules.md]: > We respect participants right

To:

> We respect our participants' right

And consider hyperlinking the word "privacy" to our privacy policy.

File: policies/rules.md [hunk 27]
```diff
@@ -0,0 +1,145 @@
...
+
+### Communication Consent and Responsibilities
+
+By participating participants agree to give organizers permission to contact them with information and communication relevant to CODEVID-19 via Slack, email, or other methods deemed appropriate by event organizers. This consent may be withdrawn at anytime be contacting contact@codevid19.com, or using the unsubscribe feature in the mailing list.
```
Comment (ajyong) [inline policies/rules.md]: By participating, <-- comma

File: policies/rules.md (hunk 20 above)
Comment (ajyong) [reply policies/rules.md]: It's a participant's

(add apostrophe to "participant")

File: policies/rules.md [hunk 28]
```diff
@@ -0,0 +1,145 @@
...
+
+Participants understand that by withdrawing their consent to communicate with organizers they will be putting themselves, their team, and their project at a competitive disadvantage. Organizers cannot be held responsible for participants receiving information supplied via email or Slack. It's participants resonsponsibiltiy to regularly check the website or Slack for updates should they choose to withdraw consent.
+
+### Competing In Other Events
+
+Participation in other hackathons and events during this period does not disqualify you from participating in CODEVID-19. As long as your project and team meet the rules and conditions you're welcome to participate in CODEVID-19 and submit your projects like other teams.
+
+Our only requirement is that all projects must join the Hackathon on FindCollabs and list their projects there before submitting.
```
Comment (ajyong) [inline policies/rules.md]: At the risk of being verbose (I like verbose rules tbh), we should re-iterate that our requirement is that they also abide by our code of conduct (link to it) and the rules outlined here.

File: policies/submissions.md [hunk 29]
```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Submitting Your Project To CODEVID-19
+---
+
+Read these carefully, otherwise we can't consider your team for prizing!
+
+### Deliverables
+
+1. REQUIRED: A link to your working app (whether that's a FindCollabs link, the website URL, or a TestFlight/Google Play opt-in URL).
```
Comment (ajyong) [inline policies/submissions.md]: Hard to say without a Markdown preview of it, but would bolding "REQUIRED" instead of all-caps be more readable?

File: policies/submissions.md [hunk 30]
```diff
@@ -0,0 +1,39 @@
...
+2. REQUIRED: A link to a screenshot we can share with voters.
+3. REQUIRED: Instructions for interacting with your app (if not included in the app itself).
+4. REQUIRED: An email address that judges can use to follow up with the team should they have questions or concerns. This will not be shared with the public, or with voters.
+5. REQUIRED: A link to your project on FindCollabs
+6. OPTIONAL: For open-source projects, we encourage a link to the source code repository (e.g. GitHub, GitLab, Bitbucket, etc.).
+
+### General Rules
+
+*   Your solution does not need to be open-source, but it must be available (upon request) for scrutiny by organizers and judges.
+*   In the spirit of hackathons, only apps that were started **on or after March 16, 00:00 UTC** will be considered for prizing.
```
Comment (ajyong) [inline policies/submissions.md]: Should this be added to the fine print?  Hmm maybe not.  There's a story to add a terms of use, which makes the fine print look quite small (and the Ontario Bar Assoc can help us draft)

**Review by ajyong — CHANGES_REQUESTED** [review]
Look at us, nitpicking.  Super duper thank you for doing this.  It brings a level of relief to my soul that I can't describe.

EDIT: @MarkBennett have we given attribution somewhere to the resources we used and modified to get us to this point?  Off the top, MLH and GLH were the ones I added to Clubhouse.

**Comment by MarkBennett** [comment]
I actually didn't source anything from them though I did read through them both. I'll see about adding a note saying thank you to them for sharing.

I'll also add a note that these rules are CC 4.0 SA.

File: policies/submissions.md (hunk 30 above)
Comment (MarkBennett) [reply policies/submissions.md]: Let's leave this were it is for now. It's explained in more detail in the `rules.md` so we're just reminding teams here. 

I'll add a link to the full rules in this section just in case teams have missed reading them in getting here.

We can do another pass after we've gotten feedback from the Bar Association but in all fairness, that will likely take them a while to do as they've probably got other priorities right now.

File: index.html (hunk 23 above)
Comment (MarkBennett) [reply index.html]: I'll udpated this next week once we've talked to judges and prepared judging resources. Until then I'm not comfortable making a specific commitment publicly. Privately, we've talked about aiming to award all prizes within a week.

File: index.html (hunk 22 above)
Comment (MarkBennett) [reply index.html:19]: A quick Google search indicates that UTF-8 is preferred these days, and HTML entities are only neccesary in cases where your editor or font doesn't support UTF-8 characters. Given that the ampersand is basically universally supported being part of the US-ASCII character set as well, I don't think we need to do this.

https://stackoverflow.com/questions/436615/when-should-one-use-html-entities

File: index.html (hunk 23 above)
Comment (MarkBennett) [reply index.html]: I've changed to the verbage you suggest.

File: policies/rules.md (hunk 25 above)
Comment (MarkBennett) [reply policies/rules.md]: You mean [like this](https://github.com/devedmonton/codevid19.com/pull/44#discussion_r399616539) @ajyong?

Just click on the time beside the comment to get a link.

File: policies/rules.md (hunk 28 above)
Comment (MarkBennett) [reply policies/rules.md]: Good catch @ajyong. I mention that projects need to meet our rules and conditions on line 127.

> As long as your project and team meet the rules and conditions you're welcome to participate in CODEVID-19 and submit your projects like other teams.

To make this clearer I'll change it to "CODEVID-19 rules and conditions".

I'll drop the "only requirement" part though, as that's not true if we're asking them to follow our rules and conditions.

File: policies/submissions.md (hunk 29 above)
Comment (MarkBennett) [reply policies/submissions.md]: Sure. I actually just ran out of time on this, but that's what I should do.

BTW if you want to see the whole file in a preview you can either click the button at the top-right of the diff to switch to rich diff mode, or open the menu at the top right of the file in the change set to "View" which will open the full file at that commit. You can then preview the entire file if that's easier to read.

File: policies/submissions.md (hunk 30 above)
Comment (MarkBennett) [reply policies/submissions.md]: I've cleaned up this whole section and clarified that people need to read the full rules. I just mentioned that you don't need to be open source but that it's encouraged to be.

**Comment by MarkBennett** [comment]
Quick update on the final rules. I've just pushed a near final draft. I copied some text from the FB hackathon and have attributed them. It's placeholder though as I want to rework it in our own words as it's not quite right for our needs and I don't want to just copy it.

Other than that, it should be ready to go for tonight's email.

**Comment by MarkBennett** [comment]
Alright, I've made another pass at the document @ajyong. Please sign off if you approve.

I also clarified some of the submission requirements and added the 

Great comments BTW!

File: policies/rules.md [hunk 31]
```diff
@@ -0,0 +1,163 @@
...
+
+### Your Solution
+
+We believe faster, more effective and more available solutions are preferred to slower, less useful and limited ones.
+
+Therefore we will be judging projects on three main criteria:
+
+1. **Effectiveness** - How well does this project address the problem it's attempting to solve?
+2. **Accessibility** - How easily are people are able to access this project?
+3. **Impact** - How many people have benefitted from this project?
```
Comment (adamelmasri) [inline policies/rules.md]: Just nit-picking but I think impact should go beyond 'number of people' (assuming the team can demonstrate what kind of impact a project can have e.g.  general economic, social, safety, etc.)

File: policies/submissions.md [hunk 32]
```diff
@@ -0,0 +1,49 @@
...
+1. **REQUIRED:** A link to your working app (the website URL, or a TestFlight/Google Play opt-in URL).
+2. **REQUIRED:** A link to a screenshot we can share with voters.
+3. **REQUIRED:** The [CODEVID-19 problem](rules.html#the-problems) you're addressing.
+4. **REQUIRED:** A short description and pitch for your app we can share on Twitter and social media (140 characters or less)
+5. **REQUIRED:** A long description and pitch for your app we can use to describe your app to voters and judges.
+6. **REQUIRED:** Instructions for interacting with your app (if not included in the app itself).
+7. **REQUIRED:** An email address that judges can use to follow up with the team should they have questions or concerns. This will not be shared with the public, or with voters.
+8. **REQUIRED:** A link to your project on FindCollabs
+9. **OPTIONAL:** For open-source projects, we encourage a link to the source code repository (e.g. GitHub, GitLab, Bitbucket, etc.).
+
```
Comment (adamelmasri) [inline policies/submissions.md:19]: If you're judging impact, allowing them an opportunity to explicitly state the type of impact they believe they'll have will make it clearer to the judges and leaves less room for inferences/assumptions.

**Review by adamelmasri — COMMENTED** [review]
Just a couple of tiny nit-picking comments

File: policies/submissions.md (hunk 32 above)
Comment (ajyong) [reply policies/submissions.md:19]: Let's move this into a separate place, I'd prefer to ship the rules as-is. I think this is a better discussion in #wg-policy.

File: policies/rules.md (hunk 31 above)
Comment (ajyong) [reply policies/rules.md]: I moved this into #wg-policy, I do agree this needs some thought and conversation around it, but I also want participants to have these rules out as-is.

File: policies/faq.md (hunk 15 above)
Comment (ajyong) [reply policies/faq.md]: Made ze update, Mandy let me know that if you're already registered and you click the invite link, Slack will automatically redirect you to our workspace, so I condensed your recommendation even further by chopping the last bit

File: policies/submissions.md (hunk 32 above)
Comment (ajyong) [reply policies/submissions.md:19]: Moved to #wg-policy

**Review by ajyong — APPROVED** [review]
LGTM, let's ship this and nitpick at a later time.  I think we've nitpicked 90%+ of the errors and the discussion from @adamelmasri has been moved to #wg-policy.

**Comment by ajyong** [comment]
https://github.com/devedmonton/codevid19.com/pull/44/commits/4818a811757802bd001c41c9f73160de1c9fa870 underscores just how error-prone manual HTML can be.  I've made the same mistake so many times in Django templates and it can be like searching for needles in a haystack.  I used https://validator.w3.org to help out.
